deep_qa/data/dataset_readers/squad_sentence_selection_reader.py

```diff
@@ -3,6 +3,8 @@
 import logging
 import os
 import random
+
+import nltk
```

Comment:
Why move it back?  Because of pylint?  You could disable the pylint warning.  I guess the real question is whether the slow import at the top matters.  Probably you would be running this script stand-alone, and not importing it, and so the slow import doesn't matter.